.github/header-checker-lint.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file configures a GitHub Bot called "License Header Lint GCF": https://github.com/apps/license-header-lint-gcf
+# The schema for this configutation file is documented at https://github.com/googleapis/repo-automation-bots/tree/main/packages/header-checker-lint#header-checker-lint.
+
+allowedCopyrightHolders:
+  - 'Google LLC'
+allowedLicenses:
+  - 'Apache-2.0'
+ignoreFiles:
+  - '**/requirements.txt'
+# If you want to ignore checking the license year, uncomment the line below.
+# ignoreLicenseYear: true # Useful when migrating in code licensed at previous years.
+sourceFileExtensions:
+  - 'cs'
+  - 'Dockerfile'
+  - 'go'
+  - 'java'
+  - 'js'
+  - 'php'
+  - 'py'
+  - 'rb'
+  - 'sh'
+  - 'tf'
+  - 'txt'
+  - 'yaml'
+  - 'yml'
```

Comment:
Let me know if I should remove this chunk:
```
# If you want to ignore checking the license year, uncomment the line below.
# ignoreLicenseYear: true # Useful when migrating in code licensed at previous years.
```
(e.g., if you think it's redundant)

Reply:
I think `txt` should be removed

Reply:
+1 we don't need copyright headers in e.g. `requirements.txt` files

Reply:
Sounds good!
I've removed both:
```
ignoreFiles:
  - '**/requirements.txt'
```
and
```
sourceFileExtensions:
  - 'txt'
```